src/pages/History.js

```diff
@@ -0,0 +1,23 @@
+import { Fragment } from 'react';
+import { Header, VideoCard } from '../components';
+import './History.css';
+
+const History = props => {
```

Comment:
you can remove this props basically You have defined props but u r not using it anywhere

Reply:
Thank for pointing it out. I'll keep it in mind.